package.json

```diff
@@ -52,7 +52,7 @@
     "@nuxt/devtools": "0.8.0",
     "@nuxtjs/i18n": "8.0.0-beta.12",
     "@rotki/eslint-config": "1.1.1",
-    "@rotki/ui-library": "0.3.0",
+    "@rotki/ui-library": "0.4.0",
```

Comment:
Need to update this again, because https://github.com/rotki/rotki.com/pull/145 still needed.